psi4/src/psi4/fnocc/df_ccsd.cc

```diff
@@ -418,6 +422,9 @@ PsiReturnType DFCoupledCluster::CCSDIterations() {
   outfile->Printf("        D1 diagnostic:                  %20.12lf\n",sqrt(eigval->pointer()[0]));
   outfile->Printf("\n");
 
+  // add D1 diagnostic to globals
+  Process::environment.globals["CC D1 DIAGNOSTIC"] = sqrt(eigval->pointer()[0]);
+
```

Comment:
We should be duplicating these into `wfn.variables` since `P::e.globals` is going to be deprecated at some point in the future